aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -267,6 +267,24 @@ In Visual Studio, the default launch profile is for IIS Express. To run the proj
 
 For apps hosted by HTTP.sys that interact with requests from the Internet or a corporate network, additional configuration might be required when hosting behind proxy servers and load balancers. For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](xref:host-and-deploy/proxy-load-balancer).
 
+<a name="ihsrtf8"></a>
+
+## Get detailed timing information with IHttpSysRequestTimingFeature
+<!--
+<xref:Microsoft.AspNetCore.Server.HttpSys.IHttpSysRequestTimingFeature> 
+-->
+[IHttpSysRequestTimingFeature](https://source.dot.net/#Microsoft.AspNetCore.Server.HttpSys/IHttpSysRequestTimingFeature.cs,3c5dc86dc837b1f4) provides detailed timing information for requests:
```

Comment:
Should you include info about a 0 value meaning the timing isn't available and that the timestamps are retrieved using QueryPerformanceCounter. All that is in the interface docs. Not sure how much of that you want to duplicate here.